presto-main/src/test/java/io/prestosql/operator/scalar/TestDateTimeFunctionsBase.java

```diff
@@ -733,6 +733,21 @@ public void testParseDatetime()
         assertFunction("parse_datetime('1960/01/22 03:04 +0500', 'YYYY/MM/DD HH:mm Z')",
                 TIMESTAMP_WITH_TIME_ZONE,
                 toTimestampWithTimeZone(new DateTime(1960, 1, 22, 3, 4, 0, 0, DateTimeZone.forOffsetHours(5))));
+
+        // The above 'YYYY/MM/DD' returns unexpected result when the month of year isn't January. The format should be 'yyyy/MM/dd'
```

Comment:
maybe just fix the test cases to use `yyyy/MM/dd` correctly?

Reply:
Updated.